### ISSUE TITLE
Model memorizes previous run

### ISSUE THREAD

**Issue by Dango233** [multimodalart/majesty-diffusion#18]
Reported by @aiartdue77.

**Comment by illtellyoulater** [comment]
Hi @Dango233 what do you mean by "model memorizes previous run" ? 
You're not saying that the simple inference is causing some kind of memory effect on the model, aren't you? :) I mean, I am asking because I with my limited knowledge in ML I don't know whether that would be even possible or not.... so yeah, just curious to understand!

**Comment by Dango233** [comment]
> 

Yes that's exactly the report I got. I can't reproduce this now locally so I'm really confused.
It might be the ema that I didn't disable in the config but not sure...

Yeah  "the simple inference is causing some kind of memory effect on the model" sounds crazy, but things happen

Just put here as a record. If everything goes on well I'll close this

**Comment by illtellyoulater** [comment]
This field will never get me bored 😅

**Comment by illtellyoulater** [comment]
If any changes actually happen inside the model, shouldn't the model file "last modified date" attribute change too to reflect that?
I don't see that happening...

**Comment by Pillazo** [comment]
Ran into this, hopefully it shows some light on what may have happened to cause it, but I was running a few tests, was working just fine, added an init_image, and ran just the 'basic settings' portion again, but it just took the image from the previous run and tried to work on that instead of starting over.

**Comment by gigaturbo** [comment]
I've got a similar phenomenon, I have parts of my previous input in the following run. If I mention a "field" for example I sometimes have it in the next run.
